src/java.base/share/classes/java/net/URL.java

```diff
@@ -956,7 +926,7 @@ void set(String protocol, String host, int port,
 
     /**
      * Returns the address of the host represented by this URL.
-     * A {@link SecurityException} or an {@link UnknownHostException}
+     * An {@link UnknownHostException}
      * while getting the host address will result in this method returning
```

Comment:
If you are doing any further edits then maybe re-flow this paragraph.

Reply:
Done. @AlanBateman could you re-review please? That's the only change.